councilmatic_core/management/commands/import_data.py

```diff
@@ -883,6 +883,7 @@ def insert_raw_memberships(self, delete=False):
                     'role': membership_json['role'],
                     'start_date': start_date,
                     'end_date': end_date,
+                    'extras': json.dumps(membership_json['extras']),
```

Comment:
If the response from the API doesn't have extras, it'll just be an empty `dict`, right?

Reply:
that's right